dirsrvtests/tests/suites/paged_results/paged_results_test.py

```diff
@@ -1126,6 +1127,8 @@ def test_multi_suffix_search(topology_st, create_user, new_suffixes):
         topology_st.standalone.restart(timeout=10)
 
         access_log_lines = topology_st.standalone.ds_access_log.match('.*pr_cookie=.*')
+        # Sort access_log_lines by op number to mitigate race condition effects. 
+        access_log_lines.sort(key=lambda x: int(re.search(r"op=(.*?) RESULT", x).group(1)))
```

Comment:
Minor nitpick, but I think we should be as specific as possible to avoid any weird issues:

```suggestion
        access_log_lines.sort(key=lambda x: int(re.search(r"op=(\d+) RESULT", x).group(1)))
```

Reply:
Good catch! The commit is amended now with that change.

Reply:
Full test runtime comes in at 7 minutes 2 seconds.